tests/integration/ceph_smoke_test.go

```diff
@@ -341,3 +344,56 @@ func (s *SmokeSuite) getNonCanaryMonDeployments() ([]appsv1.Deployment, error) {
 	}
 	return nonCanaryMonDeployments, nil
 }
+
+// Smoke Test for Webhooks
+func (s *SmokeSuite) TestWebhooks() {
+	logger.Info("Create webhook Smoke Test")
+	ctx := context.TODO()
+	cephCRList, err := s.k8sh.RookClientset.CephV1().CephClusters(s.settings.Namespace).List(ctx, metav1.ListOptions{})
+	require.NoError(s.T(), err)
+	cephCR := cephCRList.Items[0]
+	cephCR.Spec.DataDirHostPath = "/var"
+
+	_, err = s.k8sh.RookClientset.CephV1().CephClusters(s.settings.Namespace).Update(ctx, &cephCR, metav1.UpdateOptions{})
+	require.Error(s.T(), err)
+
+	pool := &cephv1.CephBlockPool{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "webhook-pool",
+			Namespace: s.settings.Namespace,
+			UID:       types.UID("c47cac40-9bee-4d52-823b-ccd803ba5bfe"),
+		},
+		Spec: cephv1.NamedBlockPoolSpec{
+			PoolSpec: cephv1.PoolSpec{
+				Replicated: cephv1.ReplicatedSpec{
+					Size: uint(1),
+				},
+				ErasureCoded: cephv1.ErasureCodedSpec{
+					CodingChunks: 1,
+					DataChunks:   1,
+				},
+			},
+		},
+	}
+
+	_, err = s.k8sh.RookClientset.CephV1().CephBlockPools(s.settings.Namespace).Create(ctx, pool, metav1.CreateOptions{})
+	require.Error(s.T(), err)
+
+	objectStore := &cephv1.CephObjectStore{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "webhook-store",
+			Namespace: s.settings.Namespace,
+		},
+		TypeMeta: metav1.TypeMeta{
+			Kind: "CephObjectStore",
+		},
+		Spec: cephv1.ObjectStoreSpec{
+			Gateway: cephv1.GatewaySpec{
+				Port:       0,
+				SecurePort: 0,
+			},
+		},
+	}
+	_, err = s.k8sh.RookClientset.CephV1().CephObjectStores(s.settings.Namespace).Create(ctx, objectStore, metav1.CreateOptions{})
```

Comment:
please include tests for subvolumegroup and radosnamespace also.

Reply:
added

Reply:
I have made the change, but I recall somewhere I saw/heard that in the import section there should be a space when the different sources of modules are imported. For example, all `github.com/...` should be in one block and `k8s.io/...` should be in a different block.
Do you have any idea about this?

Reply:
import order https://zchee.github.io/golang-wiki/CodeReviewComments/#imports https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#imports